openml/runs/functions.py

```diff
@@ -924,6 +931,8 @@ def _list_runs(
         api_call += "/flow/%s" % ','.join([str(int(i)) for i in flow])
     if uploader is not None:
         api_call += "/uploader/%s" % ','.join([str(int(i)) for i in uploader])
+    if study is not None:
```

Comment:
Could you please add a unit test to `tests.test_runs.test_run_functions.py`?

Reply:
same as previous somment (should be tested as well)

Reply:
line 153 of test_study_functions.py